vars/CentOS_10.yml

```diff
@@ -1,6 +1,8 @@
 # SPDX-License-Identifier: GPL-3.0-only
 ---
+__postfix_default_map_type: "lmdb"
```

Comment:
So EL 10 uses lmdb map type, what other oses use?

Reply:
oops - wasn't supposed to merge this

Reply:
nevermind - this is the one I closed

> So EL 10 uses lmdb map type, what other oses use?

They all use .db, which I believe is the old berkeley db format (which is deprecated now in favor of lmdb)